app/src/main/java/com/android/wildex/ui/social/FriendScreen.kt

```diff
@@ -134,7 +140,12 @@ fun FriendScreen(
               Column(modifier = Modifier.fillMaxSize()) {
                 if (uiState.isCurrentUser) {
                   CurrentUserFriendScreenContent(
-                      selectedTab, setSelectedTab, friendScreenViewModel, uiState, onProfileClick)
+                      selectedTab,
+                      setSelectedTab,
+                      friendScreenViewModel,
+                      uiState,
+                      onProfileClick,
+                      userIndex)
```

Comment:
Part of the the exclude suggestion in the general comment

```suggestion
                  CurrentUserFriendScreenContent(
                      selectedTab,
                      setSelectedTab,
                      friendScreenViewModel,
                      uiState,
                      onProfileClick,
                      userIndex,
                      userId)
```